lib/krikri/index_querier.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri
+  # Queries the Search Index
+  class IndexQuerier
```

Comment:
I'm not sure if this addresses the need to return a new instance of `Blacklight::SolrResponse`, but it'll help with the first two calls probably?